migrate.py

```diff
@@ -208,10 +286,43 @@ def fail(msg: str) -> NoReturn:
             else:
                 new_paths = art_paths
 
+            if windowsToLinuxPath:
+                new_paths = new_paths.replace("\\", "/")
+            if linuxToWindowsPath:
+                new_paths = new_paths.replace(
+                    "/",
+                    "\\",
+                )
+
             cursor.execute(
                 "UPDATE album SET embed_art_path = ?, paths = ? WHERE id = ?",
                 (new_embed_path, new_paths, id),
             )
+        # Update albums image_files column
+        albums = cursor.execute("SELECT id, image_files FROM album").fetchall()
+        for id, image_files, *_ in albums:
+            if image_files:
```

Comment:
As a simplicity, (I know other code has wrong pattern, will patch that soon), could simplify this by doing
```python
if not image_files:
    continue
```
(since no need to actually make any changes)

Reply:
I am confused on this one. Yea, I'm not changing anything. But could I just do:
```
else:
    continue
```
I think this would be just the same.